doc/api/http.md

```diff
@@ -652,7 +669,7 @@ added: v0.1.0
 * `response` {http.ServerResponse}
 
 Emitted each time there is a request. Note that there may be multiple requests
-per connection (in the case of keep-alive connections).
+per connection (in the case of HTTP Keep-Alive connections).
```

Comment:
Yes, sorry, I am saying this is the correct usage, in contrast to a number of other places where I commented that it wasn't.